### ISSUE TITLE
The herustic to choose pretrain weight?

### ISSUE THREAD

**Issue by MarSaKi** [cshizhe/VLN-DUET#13]
Hi, thanks for your great work!
I found the performance of DUET on REVERIE dataset highly depends on the initial pretrained weight. 
For example, ckpt20000 and ckpt40000 result in different finetuning performance, though they have similar performance on pretraining tasks (MLM, SAP, OG). Do you have any heuristic to choose the best pretrained weight (which iteration) to initialize DUET?

Best,

**Comment by cshizhe** [comment]
Hi, I run an online testing with the pretained ckpts and select the checkpoint with highest navigation performance.

**Comment by HonsenZhang** [comment]
> Hi, I run an online testing with the pretained ckpts and select the checkpoint with highest navigation performance.

@cshizhe Hi, could you please tell me what exactly dose the "navigation performance" mean? Is it the global/local/fused accuracy of SAP? or something else? Thanks
